client/landing/stepper/declarative-flow/trial-wooexpress-flow.ts

```diff
@@ -76,10 +78,9 @@ const wooexpress: Flow = {
 		const storeType = useSelect( ( select ) => select( ONBOARD_STORE ).getStoreType() );
 
 		const flowProgress = useSiteSetupFlowProgress( currentStep, intent, storeType );
```

Comment:
I'm wondering why we need the `flowProgress` piece for this flow (that's the large full-width progress bar at the top of the screen, right?), since it's not a flow with steps that the user can interact with or control. The current behavior is a bit wonky:

https://user-images.githubusercontent.com/2124984/211892054-f0dcd45e-dfd8-4dfe-a20e-76df993beb63.mov

Progress bar starts complete, goes back to the beginning, goes back to complete.

We also show a second progress bar during one of the `processing` steps, and a different spinner effect with the pulsing dots under the text. The overall experience is confusing :D

Reply:
Agreed that the differing spinners is confusing but that's a known issue. We have the same problem with the tailored ecommerce flow.

As far as the progress bar itself, I can see if I can hide it entirely.